designs/v1beta1-full-changelist.md

```diff
@@ -0,0 +1,200 @@
+# Karpenter v1beta1 Full Change List
+
+This document formalizes the [v1beta1 laundry list](https://github.com/aws/karpenter/issues/1327) into a full change list of items that are coming in the migration from the `v1alpha5` APIs of Karpenter to `v1beta1`. This document purely describes the necessary changes and the rationale behind the changes. For the high-level overview of the API specs, view the [Karpenter v1beta1 Graduation](./v1beta1-api.md) design doc.
+
+### Update Kind/Group Naming
+
+As part of the bump to v1beta1, to allow the v1alpha5 APIs to exist alongside the v1beta1 APIs while users go through a migration process, the following renames are being proposed:
+
+1. API Group Renames
+    1. `karpenter.k8s.aws` → `compute.k8s.aws`
+2. Kind Renames
+    1. `Provisioner` → `NodePool`
+    2. `Machine` -> `NodeClaim`
```

Comment:
How accurate is the “claim” analogy? It's not like I can adopt an existing running instance (cf. PersistentVolumeClaim, ResourceClaim), right? 

AIUI Karpenter's internal model is that it creates a Machine and then drives towards the desired state (healthy Node) by setting up instances.
You could even imagine a NodePool that has manual [ClickOps](https://www.lastweekinaws.com/blog/clickops/) provisioning with a team that should know better following some runbook to bring up a new node using the `poor-life-choice` node class.

Reply:
I'm half convinced, but I still prefer Machine.